autogpt_platform/backend/backend/copilot/db_test.py

```
@@ -2,8 +2,9 @@
 
 from __future__ import annotations
 
+from contextlib import asynccontextmanager
 from datetime import UTC, datetime
-from typing import Any
+from typing import Any, AsyncIterator
```

Ruff (0.16.1)

[warning] Import from `collections.abc` instead: `AsyncIterator`

Import from `collections.abc`

(UP035)